app/extension/maven-plugin/src/main/java/io/syndesis/extension/maven/plugin/GenerateMetadataMojo.java

```diff
@@ -581,26 +577,24 @@ private void generateAtlasMapInspections() throws MojoExecutionException {
             Optional<String> specs = generateInspections(actionId, dataShape.get().getKind(), dataShape.get().getType());
             if (specs.isPresent()) {
                 String inspection = specs.get();
-                if (Objects.equals(dataShape.get().getMetadata().get("compression"), "true")) {
-                    ByteArrayOutputStream bos = new ByteArrayOutputStream();
-
-                    try(GZIPOutputStream os = new GZIPOutputStream(bos)) {
-                        os.write(inspection.getBytes(StandardCharsets.UTF_8));
-                        os.finish();
-
-                        inspection =  Base64.getEncoder().encodeToString(bos.toByteArray());
-                    }
-                }
-
-                DataShape.Builder builder = new DataShape.Builder();
-                builder.createFrom(dataShape.get());
-                builder.specification(inspection);
-
-                for (DataShape variant : dataShape.get().getVariants()) {
-                    generateInspections(actionId, Optional.of(variant)).ifPresent(builder::addVariant);
-                }
-
-                return Optional.of(builder.build());
+                DataShape.Builder builder = new DataShape.Builder()
+                                                    .createFrom(dataShape.get())
+                                                    .specification(inspection);
+
+                builder.variants(dataShape.get().getVariants()
+                                                 .stream()
+                                                 .map(variant -> {
+                                                     try {
+                                                         return generateInspections(actionId, Optional.of(variant));
+                                                     } catch (Exception e) {
```

Comment:
Bit ugly with catching any Exception, might cause bugs to be unnoticed...

Reply:
true, at least log the error as a warning